docs/open-source-project/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,631 @@
+- Media Controller
+  - Management of content season, episode, and resolution metadata has been added.
+  - Support for getting and counting playlists has been updated.
+  - Support for counting search conditions has been added.
+- Image Util
+  - image_util_calculate_buffer_size() has been deprecated.
+  - Sync transform API set (convert, crop, resize, and rotate) has been added.
+- Thumbnail Util
+  - Extract thumbnails even if requested size exceeds the source size.
+- Camera
+  - Change buffer sharing mechanism to improve buffer protection.
+  - Support user allocated buffer for zero copy preview.
+- Recorder
+  - Change buffer sharing mechanism to improve buffer protection.
+- Audio IO
+  - Support for applying VoIP latency in case of VoIP audio type.
+- PulseAudio
+  - Support for triggering audio ducking effect by recording stream also.
+  - Support for applying volume type to loopback stream.
+  - Support for getting the PID of the latest stream.
+  - Support for individual volume has been added.
+- Audio HAL
+  - API to set volume ratio to control H/W volume according to each stream type has been added.
+- C# API
+  - Common
+    - Deprecated P/Invoke functions have been changed to recommended functions.
+  - AudioManager
+    - API to check whether any stream from the current AudioStreamPolicy is actually using the device has been added.
+  - ImageUtil
+    - API set to calculate buffer size has been deprecated.
+  - MediaContent
+    - API set to control storage information has been deprecated.
+    - The possibility of MediaContent DB inconsistency has been removed.
+  - MediaTool
+    - API set to set max bps and duration of MediaTool has been added.
+  - MediaPlayer
+    - API set to control AudioPitch has been added.
+    - The precondition of GetDownloadProgress API has been changed.
+    - Description of Preparing state restriction has been added.
+    - The range and default value related BufferingTime have been changed.
+  - Metadata
+    - API to set composer has been added.
+
+### Fixes
+
+- Media Content
+  - media_info_insert_to_db() returns the updated media information. In the past, this API returns the media information without verifying that it was up-to-date.
+  - media_face_create() support only for the images.
+- Sound Manager
+  - Timeout issue of acquiring/releasing focus in focus watch callback when the previous request is in progress has been fixed.
+- PulseAudio
+  - Invalid device of virtual stream issue in case of device connection change has been fixed.
+- C# API
+  - Camera
+    - Block issue caused by calculating preview buffer size has been fixed.
+  - MediaController
+    -  The playlist missing in P/Invoke issue has been fixed.
+  - MediaPlayer
+    - SetPlayPosition() block issue with MediaStreamSource has been fixed.
+- Multimedia Daemon
+  - Murphy
+    - Fix the launching failure issue with PIDFile.
+  - Mm-resource-manager
+    - Fix the issue of GDBus method related with usb detachable.
```

Comment:
Suggest rephrasing to:

The issue of the GDBus method related with the USB detachable has been fixed.

Reply:
will accept